### PR TITLE
feat: add planning error monitor package

### PR DESCRIPTION
**Pull request by taikitanaka3**
Related Issue
#1

Related Link
This PR depends on #4

Description
Add map packages from https://github.com/tier4/AutowareArchitectureProposal.iv

How to review
Check the following items

planning_error_monitor

### CODE REVIEW

File: planning/planning_error_monitor/launch/planning_error_monitor.launch.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<launch>
+  <!--final output-->
+  <arg name="input/trajectory" default="/planning/scenario_planning/trajectory" />
+  <!--avoidance-->
+  <!--arg name="input/trajectory" default="/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory" /-->
+  <!--surround obstacle check-->
+  <!--arg name="input/trajectory" default="/planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/trajectory" /-->
+  <!--obstacle stop-->
+  <!--arg name="input/trajectory" default="/planning/scenario_planning/lane_driving/trajectory" /-->
```
Comment (1222-takeshi) [inline planning/planning_error_monitor/launch/planning_error_monitor.launch.xml:9]: Could you remove or turn off the comment o　ut?

Comment (taikitanaka3) [reply planning/planning_error_monitor/launch/planning_error_monitor.launch.xml:9]: @1222-takeshi 
this is the error detection package and this node analyze different layer of trajectory so I added  with comment out in order to refer each trajectory easily.

@kenji-miyake 
Do you have any good idea to have optional topics as input?

Comment (kenji-miyake) [reply planning/planning_error_monitor/launch/planning_error_monitor.launch.xml:9]: Hmm, I think you can add an option `trajectory_type` or so.

```
// pseudo code
if trajectory_type == scenario_planning
    /planning/scenario_planning/trajectory
else if trajectory_type == avoidance
    /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory
else if ...
end
```

Comment (kenji-miyake) [reply planning/planning_error_monitor/launch/planning_error_monitor.launch.xml:9]: Or just adding some comments would be fine, I feel.

Comment (taikitanaka3) [reply planning/planning_error_monitor/launch/planning_error_monitor.launch.xml:9]: @kenji-miyake 
is it possible in .xml files to have condition?

Comment (kenji-miyake) [reply planning/planning_error_monitor/launch/planning_error_monitor.launch.xml:9]: Like this.
https://github.com/tier4/AutowareArchitectureProposal_launcher/blob/2af084219d0fae7effa46ef1277bb8f2082f19c6/system_launch/launch/system.launch.xml

Comment (taikitanaka3) [reply planning/planning_error_monitor/launch/planning_error_monitor.launch.xml]: @1222-takeshi @kenji-miyake 
fixed like this 99bb00b

**Review by 1222-takeshi — APPROVED** [review]
LGTM

**Pull request merged**